README.md

```diff
@@ -96,6 +92,30 @@ The text placed between a map key and a collection value. The keyword :line will
 
 If set to a positive number, then lists will only render at most the first n elements. This can help prevent unintentional realization of infinite lazy sequences.
 
+### Using CIDER alongside Ultra
+
+Projects which extend the Clojure REPL will conflict with each other; a middleware which modifies REPL print output will break the assumption of another middleware expecting unmodified Clojure output. Specifically, Ultra and CIDER collide on certain test result values ([#79](https://github.com/venantius/ultra/issues/79)).
+
+Use either CIDER or Ultra, but not both. Configure `cider-jack-in` to skip the Leiningen user profile, and therefore skip using Ultra, in .emacs:
+
+```emacs
+; Skip :user section of ~/.lein/profiles.clj when using cider-jack-in.
+(setq cider-lein-parameters
+      "with-profile -user repl :headless :host localhost")
+```
+
+If you have a lein user profile intended to alter CIDER's behavior, consider these options:
```

Comment:
I just noticed that the README says "Leiningen" throughout the doc and not "lein." A trivial change if you'd like strict consistency.